code/game/gamemodes/blob/theblob.dm

```diff
@@ -1,19 +1,23 @@
 //I will need to recode parts of this but I am way too tired atm
 /obj/structure/blob
 	name = "blob"
+	desc = "Some blob creature thingy"
 	icon = 'icons/mob/blob.dmi'
+	anchored = TRUE
```

Comment:
is it needed to remove the Density and opacity values for the blob and not simply just set the current value to a true false rather then a numeric? And how is it related to the timer PR?

Reply:
In this case, both the [`density`](http://www.byond.com/docs/ref/#/atom/var/density) and [`opacity`](http://www.byond.com/docs/ref/#/atom/var/opacity) variables are already set to `0`/`FALSE` by default. Since I was editing the `/obj/structure/blob`'s variables already, I didn't see any harm in removing those two in order to clean up a bit since they're just redundant.
![Code_n7DdXRYEJX](https://user-images.githubusercontent.com/57483089/132930755-4bedce25-b7e4-4065-bc61-17ce6c37c4f9.png) ![Code_m6bQB09vFA](https://user-images.githubusercontent.com/57483089/132930756-c14d75a0-d72f-4d05-a38e-75c383968f37.png)